src/Firehose.Web/Firehose.Web.csproj

```diff
@@ -201,14 +201,12 @@
     <Compile Include="Authors\JasonDeBoever.cs" />
     <Compile Include="Authors\JeffersonBalivo.cs" />
     <Compile Include="Authors\JeromeGiacomini.cs" />
-    <Compile Include="Authors\JesusAngulo.cs" />
```

Comment:
I'd remove the files as well

Reply:
Yeah was in doubt about that.. Will remove them!

Reply:
Done